Update NU5131.md (#1733)
NuGet/docs.microsoft.com-nuget: docs/reference/errors-and-warnings/NU5131.md

</references>
```

> This works for projects using the package with `packages.config`, but to work as intended with `PackageReference`, the package must also contain the following files:

Reviewer: What do you think of something like this?

```suggestion
This package will not work as intended when using `PackageReference`. To fix this, the package must also contain the following files:
```

Reviewer: Is there a link that explains what `PackageReference` is? If so, please add links
